templates/views/events.html

```diff
@@ -36,16 +37,32 @@ <h4 class="event_topic1">Upcoming Events</h4>
 
             </div>
         </div>
+        {% if data.upcomingEvents|length %}
+        {% for event in data.upcomingEvents %}
+        {% if data.upcomingEvents.indexOf(event) %  2 !== 0 %}
         <div class="row row_padding">
-            <div class="col-md-5 col-lg-5 col-sm-5 event_side_upcoming">
+            <img class="col-md-5 col-lg-5 col-sm-5 event_side_past2" src="/images/event_image.png" class="img-fluid"
```

Comment:
Hi @OdunayoOkebunmi 
Instead of `src="/images/event_image.png"`, I think each event should have its own unique image.
That way, it would be `src={{event.image}}` instead of the specified image.